<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrossDlg</class>
 <widget class="QWidget" name="CrossDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CrossDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{color: rgb(255, 255, 255);
background-color: rgb(15, 21, 42);
}
QPushButton{
background-color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>256</width>
     <height>320</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>320</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>320</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 85, 0);
border-width: 2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>240</width>
     <height>256</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>256</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>256</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MyHistLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>240</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_red">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_blue">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>430</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_green">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_yellow">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyHistLabel</class>
   <extends>QLabel</extends>
   <header>myhistlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
